### PR TITLE
[UI Improvement] Save creator's e-mail address to cookie

### PR DESCRIPTION
**Pull request by tattsun**
At silence UI,
* Alertmanager saves creator's email address to cookie.
* Alertmanager automatically fill it from cookie.

Thanks.

### CODE REVIEW

File: ui/app/index.html [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <head>
 	<script src="lib/jquery.min.js"></script>
 	<script src="lib/moment.min.js"></script>
+  <script src="lib/js.cookie.js"></script>
```
Comment (fabxc) [inline ui/app/index.html]: indentation is off

**Comment by fabxc** [comment]
That seems good usability-wise.

**Comment by tattsun** [comment]
@fabxc Thanks for reviewing! Please check new commit I pushed now.

**Comment by brancz** [comment]
LGTM, wdyt @fabxc ?

**Comment by grobie** [comment]
What's the reason to store the email in a cookie instead of the browser local storage?

**Comment by tattsun** [comment]
@grobie, ah, it's simpler and better to use local storage. I fixed it to use local storage.

**Comment by fabxc** [comment]
Yea, was thinking the same. Then thought that cookies are proven and reliable. But I suppose local storage is more ubiquitous and well working than when I worked with it last time about 5 years ago :)

**Comment by fabxc** [comment]
Looks good, thank you! 👍

**Comment by brancz** [comment]
I guess both would have worked, but yes local storage is the nicer solution.

**Pull request merged**